src/components/chat-item/chat-item-form-items.ts

```diff
@@ -49,17 +49,19 @@ export class ChatItemFormItemsWrapper {
       classNames: [ 'mynah-chat-item-form-items-container', ...(this.props.classNames ?? []) ],
       children: this.props.chatItem.formItems?.map(chatItemOption => {
         let chatOption: Select | RadioGroup | TextArea | Stars | TextInput | Checkbox | FormItemList | undefined;
-        let label: ExtendedHTMLElement | string = `${chatItemOption.mandatory === true ? '* ' : ''}${chatItemOption.title ?? ''}`;
+        let label: ExtendedHTMLElement | string = `${chatItemOption.mandatory === true && chatItemOption.title !== undefined ? '* ' : ''}${chatItemOption.title ?? ''}`;
```

Comment:
Rather than interpreting the title (eg: "if title is empty"), would it be reasonable to introduce a new optional boolean flag `hideIcon` (general, could apply elsewhere) or `hideMandatoryIcon` (more specific to use of `mandatory` property)? This property would be treated as false when not present. This way you can still allow for some future element without a title to still be decorated with the mandatory icon.

Reply:
Yeah I was thinking of doing the boolean flag approach initially but I wanted to avoid adding a new flag. But I can see how the behavior might be confusing/unexpected. Let me give it another try